strftime.js

```diff
@@ -595,10 +595,8 @@
             else
                 weekday--;
         }
-
-        var firstDayOfYearUtc = Date.UTC(date.getFullYear(), 0, 1),
-            dateUtc = Date.UTC(date.getFullYear(), date.getMonth(), date.getDate()),
-            yday = Math.floor((dateUtc - firstDayOfYearUtc) / 86400000),
+        var firstDayOfYear = new Date(date.getFullYear(), 0, 1),
```

Comment:
Do we need to add the tz offset from `date` to `firstDayOfYearUtc`? This might be a bug. In some timezones the bug wouldn't manifest, but in others it will.